osu.Game.Rulesets.Tau/Mods/TauModInverse.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using osu.Framework.Graphics;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Tau.Objects;
+using osu.Game.Rulesets.Tau.Objects.Drawables;
+using osu.Game.Rulesets.Tau.UI;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Tau.Mods
+{
+    public class TauModInverse : Mod, IApplicableToDrawableHitObject, IApplicableToDrawableRuleset<TauHitObject>
+    {
+        public override string Name => "Inverse";
+        public override string Acronym => "IN";
+        public override ModType Type => ModType.DifficultyIncrease;
+        public override string Description => @"Hit objects will come outside of the playfield.";
+        public override double ScoreMultiplier => 1.2;
+        public override Type[] IncompatibleMods => new[] { typeof(TauModHidden), typeof(TauModFadeIn) };
+
+        public void ApplyToDrawableHitObject(DrawableHitObject drawable)
+        {
+            drawable.ApplyCustomUpdateState += (drawableObject, state) =>
+            {
+                switch (drawableObject)
+                {
+                    case DrawableSliderHead head:
+                        applyInverseToBeat(head, state, -0.484f, -0.984f);
+
+                        break;
+
+                    case DrawableBeat beat:
+                        applyInverseToBeat(beat, state, -0.516f);
+
+                        break;
+
+                    case DrawableHardBeat hardBeat:
+                        var hardBeatObject = hardBeat.HitObject;
+
+                        hardBeat.ClearTransforms();
+                        hardBeat.HitScale = .75f;
+                        hardBeat.MissScale = .9f;
+
+                        using (hardBeat.BeginAbsoluteSequence(hardBeatObject.StartTime - hardBeatObject.TimePreempt))
+                        {
+                            hardBeat.ResizeTo(2);
+                            hardBeat.ResizeTo(1, hardBeatObject.TimePreempt);
+                        }
+
+                        break;
+
+                    case DrawableSlider slider:
+                        slider.ApplyInverseChanges();
+
+                        break;
+                }
+            };
+        }
+
+        private void applyInverseToBeat(DrawableBeat beat, ArmedState state, float finalDistance, float startingDistance = -1)
+        {
+            var box = beat.Box;
+            var hitObject = beat.HitObject;
+
+            beat.HitDistance = .1f;
```

Comment:
This should use the negative of the existing value. To avoid inconsistencies when that value changes.